presto-main/src/test/java/com/facebook/presto/execution/TestTaskExecutor.java

```diff
@@ -115,7 +115,7 @@ public void test()
         }
     }
 
-    @Test
+    @Test(invocationCount = 10000)
```

Comment:
I think multiple runs are necessary for this test so that we could catch the race issue.